src/App.vue

```diff
@@ -146,13 +146,14 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
     // Evaluate name request pre conditions
     const nameRequest = await this.evaluateNRPreconditions()
     if (nameRequest && nameRequest.nrNum && nameRequest.isConsumable) {
-      // TODO: Handling different NR Formats
+      // TODO: Handling different NR Formats & accept the EntityType dynamically once Namex provides it.
       // Will probably change once we proxy through legal API with a consistent format
       this.setNameRequestState(
-        { nrNumber: nameRequest.nrNum.replace('NR ', ''),
-          entityType: nameRequest.requestTypeCd,
+        { nrNumber: nameRequest.nrNum,
+          entityType: EntityTypes.BCOMP,
           filingId: null })
       this.setCurrentDate(this.dateToUsableString(new Date()))
+      this.onEntityTypeChanged(this.stateModel.nameRequest.entityType)
```

Comment:
You shouldn't need this. Perhaps you need a deep watch on line 283?

Reply:
Interesting, i'll look into the deep watch.

Reply:
As it was, it wasn't updating.

Reply:
* Fixed, the reference to store was incorrect.